types/react-native/index.d.ts

```diff
@@ -42,6 +42,7 @@
 //                 Marcel Lasaj <https://github.com/TheWirv>
 //                 Alexey Molchan <https://github.com/alexeymolchan>
 //                 Alex Brazier <https://github.com/alexbrazier>
+//                 Arafat Zahan <kuasha420@gmail.com>
```

Comment:
CI is failing because this should be a GitHub user profile link. Also, are you aware that this means you are signing up to be a maintainer of these types and you will receive pings for every issue/PR for the RN types?

Reply:
Wow, didn't realize that. While I wouldn't mind getting pinged, can I submit PR without adding myself in the list? 

Cheers

Reply:
Yep, absolutely 👍

The name of the header field isn't great imo, it makes it seem as if it's just a list of people that contributed to the file 😄

Reply:
Thanks for clearing that up. I've fixed my profile link. Cheers